torchdet3d/utils/utils.py

```diff
@@ -154,7 +154,7 @@ def _remove_prefix(key, prefix):
     matched_layers, discarded_layers = [], []
 
     for k, v in state_dict.items():
-        k = _remove_prefix(k, 'module')
+        k = extra_prefix + _remove_prefix(k, 'module')
```

Comment:
Do the 21k pretrained weights for MNV3  contain prefixes like `model.module`?  Why we can't just add one more line `_remove_prefix(k, 'model')`?

Reply:
`model.` is not presented in snapshot, so there is nothing to remove. At the same time this prefix is contained in the model state dict keys, which we can't change.